wgpu-hal/src/metal/adapter.rs

```diff
@@ -807,6 +807,12 @@ impl super::PrivateCapabilities {
                 None
             },
             timestamp_query_support,
+            supports_simd_scoped_operations: family_check
+                && (device.supports_family(MTLGPUFamily::Metal3)
+                    || device.supports_family(MTLGPUFamily::Mac2)
+                    || device.supports_family(MTLGPUFamily::Apple7)
+                    || device.supports_family(MTLGPUFamily::Apple8)
+                    || device.supports_family(MTLGPUFamily::Apple9)),
```

Comment:
```suggestion
                    || device.supports_family(MTLGPUFamily::Apple7)),
```

All `Apple8+` support `Apple7`.